scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/tasks/read_group_vars.yml

```diff
@@ -93,3 +110,32 @@
   no_log: true
 
 - debug: var=ardana_group_vars_key_api
+
+- set_fact:
```

Comment:
Can you name those set_fact tasks?

Also, why you are using `no_log` ?

If those variables are only needed by keystone tests, can you move them to a different file and use `import_tasks` to run those tasks only for the keystone tests?